defaults/main.yml

```diff
@@ -17,6 +17,10 @@
 #
 # Should the shared memory be configured by the ``debops.sysctl`` role?
 sysctl__shared_memory_configure: True
+                              if not (ansible_virtualization_role == "guest"
```

Comment:
Seems like you forgot something there (Jinja opening sequence `'{{`)?

Reply:
O my, indeed missed the Jinja start curlies. Strange the playbook did run without complaining.

Reply:
Wired. That is invalid YAML.